src/ext.js

```diff
@@ -393,6 +393,90 @@ export default function ext({ translator }) {
                 },
               },
             },
+            borders: {
+              type: 'items',
+              grouped: true,
+              translation: 'properties.border',
+              items: {
+                useBorders: {
+                  type: 'items',
+                  items: {
+                    useBorders: {
+                      ref: 'style.border.isUsed',
+                      type: 'boolean',
+                      translation: 'properties.border.use',
+                      component: 'switch',
+                      defaultValue: false,
+                      options: [
+                        {
+                          value: true,
+                          translation: 'properties.on',
+                        },
+                        {
+                          value: false,
+                          translation: 'properties.off',
+                        },
+                      ],
+                    },
+                  },
+                },
+                borderSettings: {
+                  type: 'items',
+                  show: data => propertyResolver.getValue(data, 'style.border.isUsed'),
+                  items: {
+                    borderRadius: {
+                      component: 'slider',
+                      translation: 'properties.border.radius',
+                      type: 'number',
+                      ref: 'style.border.radius',
+                      min: 0,
+                      max: 100,
+                      step: 1,
+                    },
+                    borderWidth: {
+                      component: 'slider',
```

Comment:
Not doing this in pixels with an input field?

Reply:
No we want to have it responsive. I will look to change the font size as well to have it responsive, but that might need some more work